src/hugr/views/sibling.rs

```diff
@@ -25,37 +26,24 @@ type FlatRegionGraph<'g> = portgraph::view::FlatRegion<'g, &'g MultiPortGraph>;
 /// used interchangeably with [`DescendantsGraph`].
 ///
 /// [`DescendantsGraph`]: super::DescendantsGraph
-pub struct SiblingGraph<'g, Root = Node, Base = Hugr>
-where
-    Base: HugrInternals,
-{
+#[derive(Clone)]
+pub struct SiblingGraph<'g, Root = Node> {
     /// The chosen root node.
     root: Node,
 
     /// The filtered portgraph encoding the adjacency structure of the HUGR.
     graph: FlatRegionGraph<'g>,
 
-    /// The rest of the HUGR.
-    hugr: &'g Base,
+    /// View onto the underlying Hugr which this graph filters
```

Comment:
Thanks for the improved doc!

Is "view" the correct term though? I would have just said "The underlying Hugr" or "reference to..."